docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -0,0 +1,106 @@
+- Per-instance limit errors are not logged anymore, to reduce resource usage when ingesters are under pressure. We encourage to use metrics and alerting to monitor them. The following metrics have been added to count the number of requests rejected for hitting per-instance limits:
+  - `cortex_distributor_instance_rejected_requests_total`
+  - `cortex_ingester_instance_rejected_requests_total`
+- The CLI flag `-validation.create-grace-period` is now enforced in the ingester too, if you've configured `-validation.create-grace-period` then make sure the configuration is applied to ingesters too.
+- The CLI flag `-validation.create-grace-period` is now enforced for exemplars too, and the `cortex_discarded_exemplars_total{reason="exemplar_too_far_in_future",user="..."}` series is incremented when exemplars are dropped because their timestamp is greater than "now + grace_period".
+- The CLI flag `-validation.create-grace-period` is now enforced in the query-frontend even when the configured value is 0. When the value is 0, the query end time range is truncated to the current real-world time.
+
+The following metrics were removed:
+
+- `cortex_ingester_shipper_dir_syncs_total`
+- `cortex_ingester_shipper_dir_sync_failures_total`
+
+The following configuration options are deprecated and will be removed in Grafana Mimir 2.12:
+
+-The CLI flag `-blocks-storage.bucket-store.index-header-lazy-loading-enabled` is deprecated, use the new configuration `-blocks-storage.bucket-store.index-header.lazy-loading-enabled`.
+-The CLI flag `-blocks-storage.bucket-store.index-header-lazy-loading-idle-timeout` is deprecated, use the new configuration `-blocks-storage.bucket-store.index-header.lazy-loading-idle-timeout`.
+-The CLI flag `-blocks-storage.bucket-store.index-header-lazy-loading-concurrency` is deprecated, use the new configuration `-blocks-storage.bucket-store.index-header.lazy-loading-concurrency`.
+
+The following configuration options that were deprecated in Grafana Mimir 2.8 are removed:
+
+- The CLI flag `blocks-storage.tsdb.max-tsdb-opening-concurrency-on-startup`.
+
+The following experimental configuration options were renamed or removed:
+
+- The CLI flag `-querier.prefer-streaming-chunks` was renamed to `-querier.prefer-streaming-chunks-from-ingesters`.
+- The CLI flag `-blocks-storage.bucket-store.chunks-cache.fine-grained-chunks-caching-enabled` was removed.
+- The CLI flag `-blocks-storage.bucket-store.fine-grained-chunks-caching-ranges-per-series` was removed.
+
+The following experimental options are now stable:
+
+- The CLI flag `-shutdown-delay`.
+- The CLI flag `-ingester.ring.excluded-zones`.
+
+The following configuration option defaults were changed:
+
+- The default value for the CLI flag `-querier.streaming-chunks-per-ingester-buffer-size` was changed from `512` to `256`.
+- The default value for gRPC clients connect timeout was set to `5s` with a default max backoff delay of `5s`.
```

Comment:
Can you mention what the previous values were too? Or were those unset/infinity?

Reply:
Done in https://github.com/grafana/mimir/pull/5873/commits/9b7a20fc11fcbf34fd7d890d111fc0579ecbd9d9. It took me a while to find the original values.